tests/test_utils/test_config.py

```diff
@@ -304,6 +305,69 @@ def test_base_variables():
         e='t.py')
 
 
+def test_env_variables():
+    with patch.dict('os.environ', {'WHOWHO': 'OK'}):
+        for file in ['tt.py', 'tt.json', 'tt.yaml']:
+            cfg_file = osp.join(data_path, f'config/{file}')
+            cfg = Config.fromfile(cfg_file)
+            assert isinstance(cfg, Config)
+            assert cfg.filename == cfg_file
+            # cfg.field
+            assert cfg.item1 == [1, 2]
+            assert cfg.item2.a == 0
+            assert cfg.item3 is False
+            assert cfg.item4 == 'test'
+            assert cfg.item5 == dict(a=0, b=1)
+            assert cfg.item6 == [dict(a=0), dict(b=1)]
+            assert cfg.item7 == dict(a=[0, 1, 2], b=dict(c=[3.1, 4.2, 5.3]))
+            assert cfg.item8 == file
+            assert cfg.item9 == dict(a=0)
+            assert cfg.item10 == [3.1, 4.2, 5.3]
+            assert cfg.item88 == 'OK'
+
+    for file in ['tt.py', 'tt.json', 'tt.yaml']:
+        with pytest.raises(Exception):
```

Comment:
```suggestion
        with pytest.raises(ValueError):
```

Reply:
Adding the match argument here will help us know why it should raise an exception.